language-server/src/main/java/org/sonarlint/languageserver/SonarLintLanguageServer.java

```diff
@@ -174,34 +205,15 @@ private UserSettings(Map<String, Object> params) {
       this.disableTelemetry = (Boolean) params.getOrDefault(DISABLE_TELEMETRY, false);
     }
 
-    private static Map getAnalyzerProperties(Map<String, Object> params) {
-      Map analyzerProperties = (Map) params.get(ANALYZER_PROPERTIES);
-      if (analyzerProperties == null) {
-        analyzerProperties = Collections.emptyMap();
+    private Map<String, String> getAnalyzerProperties(Map<String, Object> params) {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Make "getAnalyzerProperties" a "static" method. ([squid:S2325](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS2325))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.sonarlint.core%3Asonarlint-core-parent&pullRequest=192&issues=AWNEQd1A6NykrOWnW1WD&open=AWNEQd1A6NykrOWnW1WD)

[ISSUE_KEY]: AWNEQd1A6NykrOWnW1WD